src/plugins/poster/poster.js

```diff
@@ -95,12 +95,15 @@ export default class PosterPlugin extends UIContainerPlugin {
   }
 
   clicked() {
-    if (!this.options.chromeless || this.options.allowUserInteraction) {
-      this.playRequested = true
-      this.update()
-      this.container.play()
+    // Let "click_to_pause" plugin handle click event if media has started playing
+    if (! this.hasStartedPlaying) {
+      if (!this.options.chromeless || this.options.allowUserInteraction) {
+        this.playRequested = true
+        this.update()
+        this.container.play()
+      }
+      return false
     }
```

Comment:
Shouldn't we always return something? I think the last `return` is missing

Reply:
@leandromoreira if you return false, then click event is not propagated  (and therefore click_to_pause plugin click is not triggered).